src/entities.spec.js

```diff
@@ -0,0 +1,52 @@
+import main from './main'
+import * as usersData from './test/users'
+import * as userData from './test/user'
+
+main.initialize({
+  host: 'http://localhost:8080'
+})
+
+const entities = main.entities()
+
+describe('toccojs', () => {
+  describe('entities', () => {
+    afterEach(() => {
+      fetch.resetMocks()
+    })
+
+    test('should list logged-in users', done => {
+      fetch.mockResponse(JSON.stringify(usersData))
+      entities.list('User').then(data => {
```

Comment:
correct url and request params should be verified as well (as in the tests for the get() method)